R/Design-methods.R

```diff
@@ -2142,20 +2131,13 @@ setMethod("simulate",
                                  model=thisModel,
                                  data=thisData,
                                  SIM=TRUE)
-                  thisDose <- NEXT$nextdose
-
-                  thisTDtargetDuringTrial<- NEXT$TDtargetDuringTrialEstimate
-
-
-                  thisTDtargetEndOfTrial<- NEXT$TDtargetEndOfTrialEstimate
-
-                  thisTDtargetEndOfTrialatdoseGrid <- NEXT$TDtargetEndOfTrialatdoseGrid
-
-                  thisCITDEOT <- list(lower=NEXT$CITFEOT[1],
-                                      upper=NEXT$CITFEOT[2])
-
-                  thisratioTDEOT <- NEXT$ratioTDEOT
 
+                  thisDose <- NEXT$next_dose_drt
+                  thisTDtargetDuringTrial<- NEXT$dose_target_drt
+                  thisTDtargetEndOfTrial<- NEXT$dose_target_eot
+                  thisTDtargetEndOfTrialatdoseGrid <- NEXT$next_dose_eot
+                  thisCITDEOT <- list(lower=NEXT$ci_dose_target_eot[1], upper=NEXT$ci_dose_target_eot[2])
```

Comment:
```suggestion
                  thisCITDEOT <- list(lower = NEXT$ci_dose_target_eot[1], upper = NEXT$ci_dose_target_eot[2])
```
etc. in other places maybe, but I guess styler should catch this

Reply:
Yap, this file is before cleanup, I have just cleaned the code around the lines I changed, but it will be anyway fully cleaned-up once we reach this file.